src/feature_annotation/score_genes_cell_cycle_scanpy/script.py

```diff
@@ -0,0 +1,28 @@
+import scanpy as sc
+import mudata as mu
+
+
+mudata = mu.read(f'{par["input"]}')
+
+par["use_raw"] = par.setdefault("use_raw", None)
+par["gene_pool"] = par.setdefault("gene_pool", None)
+
+s_genes = [x.strip() for x in open(par["s_genes"])]
+g2m_genes = [x.strip() for x in open(par["g2m_genes"])]
+
+if par["gene_pool"]:
+    par["gene_pool"] = [x.strip() for x in open(par["gene_pool"])]
+
+output = sc.tl.score_genes_cell_cycle(
```

Comment:
Could you check that the genes that are specified exist in the input data?

Reply:
Can add an assert, but actually scanpy is handling that already - if there are any genes that are in the gene pool but nog in the input, it will raise a warning, but will run the scoring on the other genes that are present

Reply:
Proposal: perhaps we could we make this an argument? By default the component will raise an error, but when the argument (for example `--allow_unknown_genes`) is passed a warning is used?